test-cases/gemini/gemini-8h-large-num-columns.yaml

```diff
@@ -20,7 +20,7 @@ gemini_cmd: "gemini -d --duration 7h --warmup 1h \
 -c 10 -m mixed -f --non-interactive \
 --cql-features normal --async-objects-stabilization-backoff 500ms \
 --replication-strategy \"{'class': 'NetworkTopologyStrategy', 'replication_factor': '3'}\" \
---max-mutation-retries 5 --max-mutation-retries-backoff 500ms \
+--max-mutation-retries 10 --max-mutation-retries-backoff 500ms \
```

Comment:
When we reach timeouts it is mostly up to 2s delay.
So, probably better to increase retry interval/backoff from 0.5s to 1-2 seconds.

I do not insist. Up to you.